src/js/index.js

```diff
@@ -46,6 +46,9 @@ function push_response(msg, callback) {
 function push_yt_response(id) {
     $('.chat').append('<div style="border-radius: 5px !important;" class="bubble you"><div id="player_container"><div class="player" id="player' + id + '"></div></div></div>')
 }
+function push_image(url,height,width){
+    $('.chat').append('<div style="border-radius: 5px !important;height: '+(height+40)+'px;width: '+(width+40)+'px;" class="bubble you"><div><img src="'+url+'" height="'+height+'" width="'+width+'"></div></div></div>');
```

Comment:
I meant to hard-code the width here (or in the CSS) rather than in the push_image call. Sorry for the confusion

Reply:
Okay, it is hardcoded.